cs/v1/CS_v1.0.0.yaml

```diff
@@ -603,96 +604,13 @@ components:
         modeOfTransport:
```

Comment:
Not sure if `modeOfTransport` is the correct name here since it is no longer a property - now it is an object. See Teams